.env

```
@@ -0,0 +1,8 @@
+EXPO_PUBLIC_STREAM_API_KEY=tk39tr6hz45e
+
+EXPO_PUBLIC_SUPABASE_URL=https://rsuoggyhcqbtmwpkqmbf.supabase.co
+
+EXPO_PUBLIC_SUPABASE_PUBLISHABLE_KEY=
+
+EXPO_PUBLIC_SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InJzdW9nZ3loY3FidG13cGtxbWJmIiwicm9sZSI6ImFub24iLCJpYXQiOjE3NTg4NDU3MjksImV4cCI6MjA3NDQyMTcyOX0.fpNwSQMDcXklTRDKJEQodc6yFI5lCe7VvVS_VotIe4U
+hayatudeen
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)